### DISCUSSION TITLE
Access to the template files after publish

### DISCUSSION THREAD

**Discussion by masoudliz** [OrchardCMS/OrchardCore#15206] [Q&A]
Hello
After publishing, there is no file in the wwwroot folder that I can use to edit the site template.
Is there a way to edit the frontend files after publishing? I mean the template file.

**Comment by Piedone** [comment]
Orchard's frontend is built completely differently. either with [themes](https://docs.orchardcore.net/en/latest/docs/glossary/#theme) or in [a decoupled way](https://docs.orchardcore.net/en/latest/docs/guides/decoupled-cms/). I suggest you start with one of the [tutorials](https://docs.orchardcore.net/en/latest/docs/resources/tutorials/).
